modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/DXPActivationKeysTable/index.js

```diff
@@ -225,6 +237,16 @@ const DXPActivationKeysTable = () => {
 					}))}
 				/>
 			</ClayTooltipProvider>
+
+			{isVisibleModal && (
+				<ModalKeyDetails
+					ACTIVATION_STATUS={ACTIVATION_STATUS}
```

Comment:
This is an external constant, so you can remove here and get directly into ModalKeyDetails component
```suggestion
					
```

Reply:
Done!